reference/5.1/Microsoft.PowerShell.Core/About/about_Ref.md

```diff
@@ -149,10 +155,96 @@ $i = 0;$iRef = 1
 
 Only the reference type's variable was changed.
 
+## Using `[ref]` as a general-purpose object holder
+
+You can also use `[ref]` as a general-purpose object holder. In this usage,
+`[ref]` is applied to a _value_ instead of a variable. Typically, the value is
+an instance of a _value type_, like a number. In most scenarios you can use a
+regular variable or parameter instead. However, this technique is useful in
+scenarios where passing an explicit value holder is undesired (for brevity) or
+not possible, such as in script-block parameter values.
+
+For example, you can use script-block parameter values to calculate the value
+of **NewName** parameter of the `Rename-Item` cmdlet. The `Rename-Item` cmdlet
+allows you to pipe items to it. The command run the script block passed to the
+**NewName** for each item in the pipeline. The script block run in a child
+scope. Modifying a variable in the caller's scope directly won't help and you
+can't pass arguments to the script block in this context.
+
+In this example, the script block passed to the **NewName** parameter
+increments the value of `$iRef` for each item in the pipeline. The script block
+creates a new name by adding a number to the beginning of the filename.
+
+```powershell
+$iRef = [ref] 0
+Get-ChildItem -File $setPath |
+    Rename-Item -NewName { '{0} - {1}' -f $iRef.Value++,$_.Name }
+```
+
+## Difference between `[ref]` and `[System.Management.Automation.PSReference]`
+
+A reference type variable is created using
+
+Even though `[ref]` is a type accelerator for
+`[System.Management.Automation.PSReference]`, they behave differently.
+
+- When you use `[ref]` to cast a variable, PowerShell creates reference object
+  that contains reference to the original instance of the variable.
+- When you use `[System.Management.Automation.PSReference]` to cast a variable,
```

Comment:
"Reference object" must be "a reference object".

Reply:
Fixed in a different PR.